signing/mcl/multisig/testData/data.go

```diff
@@ -0,0 +1,81 @@
+package testData
+
+import (
+	crypto "github.com/multiversx/mx-chain-crypto-go"
+)
+
+// PredefinedTest defines the data used for testing
+type PredefinedTest struct {
+	TestName      string
+	Message       string
+	ExpectedError string
+}
+
+// predefinedKOSKAggregateSignaturesTests defines the scenarios for testing the AggregateSignatures for KOSK
+var predefinedKOSKAggregateSignaturesTests = []PredefinedTest{
```

Comment:
these structures at lines 15, 24, 33, 42 are very similar (duplicated). We can keep only one definition (rename it to something more generic) and use it for the different test vectors.